src/HashedDot.hs

```diff
@@ -234,8 +235,9 @@ extractPoly pe@(PolyEnv (_, nbidx2varIdx) _envExpr) inputs e n =
             Just
                 ( c
                 , ( I.foldr (+) 0 vars -- total degree of all variables
-                  , vars)){-
+                  , vars {-
```

Comment:
What happened here?

Reply:
LOL it moves the closed parentheses after the comment, but not too bad I think.

Reply:
It must think they should be there...